# TestNumber: used by execute_asap to send commands
time stamp
int32 test_number
string role
